src/commands/Unban.ts

```diff
@@ -101,8 +101,7 @@ export type DraupnirUnbanCommandContext = {
 };
 
 export const DraupnirUnbanCommand = describeCommand({
-  summary:
-    "Removes an entity from a policy list. If the entity is a glob, then the flag --true must be provided to unban users matching the glob from all protected rooms.",
+  summary: "Removes an entity from a policy list.",
```

Comment:
Ah, now here this is problematic, we've not realised that `--true` had dual purposes:

* The first is wholly terrible, which was to confirm that you wanted to unban users from all rooms
* The second isn't so bad, which is to make sure you understand you are using a glob. Similar to the kick command's `glob` option.

We probably do want that `glob` option to be there and we can pretty much copy what the kick command does there. What do you think?

Reply:
Ok nah, that's wrong. Because the whole point of accepting globs is to remove policy rules, not unban users matching globs.

Reply:
The message that the command sends to the management room afterwards unbans users matching a glob only because that's what it needs to do when a glob policy rule is removed